modules/openxr/doc_classes/OpenXRHand.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="OpenXRHand" inherits="Node3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
 	<brief_description>
-		Node supporting finger tracking in OpenXR.
+		Node supporting hand and finger tracking in OpenXR.
 	</brief_description>
 	<description>
-		This node enables OpenXR's hand tracking functionality. The node should be a child node of an [XROrigin3D] node, tracking will update its position to where the player's actual hand is positioned. This node also updates the skeleton of a properly skinned hand model. The hand mesh should be a child node of this node.
+		This node enables OpenXR's hand tracking functionality. The node should be a child node of an [XROrigin3D] node, tracking will update its position to where the player's actual hand is positioned. This node also updates the skeleton of a properly skinned hand or avatar model.
```

Comment:
I think we should add some clarity here of the behaviour that:
- the OpenXRHands node will be automatically placed at the hands palm location
- if one of the hand bones is the root node, the skeleton will be placed relative to the hand palm location and the hand mesh and skeleton should be children of the OpenXRHands node.
- if the hand bones are part of a full skeleton the root of the hand will keep its location with the assumption IK is used to position the hand and arm.

The current description here only covers the first two points and kind of indicate the 3rd isn't correct.

Reply:
I updated the description to contain the clarifying comments.

Reply:
Looks good!